packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -64,6 +64,79 @@ abstract class InteractiveInkFeature extends InkFeature {
     _color = value;
     controller.markNeedsPaint();
   }
+
+  /// Draws an ink splash or ink ripple on the passed in [Canvas].
+  ///
+  /// The [transform] argument is the [Matrix4] transform that typically
+  /// shifts the coordinate space of the canvas to the space in which
+  /// the ink circle is to be painted.
+  ///
+  /// [center] is the [Offset] from origin of the canvas where center
+  /// of the circle is drawn
+  ///
+  /// Using [paint] you can specify properties like color, strokewidth, colorFiler, etc
+  /// for painting the circle
+  ///
+  /// [radius] is the radius of circle to be drawn on canvas
+  ///
+  /// [clipCallback] is required by ink effects to obtain the rectangle for the effect,
+  /// if [clipCallback] is null, no clipping is performed and only the ink circle is painted
+  ///
+  /// [customBorder] is a [ShapeBorder] which can be used if you want to clip
+  /// something other than a simply a [Rect] or [RRect]
+  ///
+  /// If [customBorder] is provided, the canvas is clipped by it. If
+  /// [customBorder] is null, canvas is clipped by the [RRect]
+  /// formed by [borderRadius] and [clipCallBack]
+  ///
+  /// [textDirection] is used if [customBorder] is provided (as [customBorder] can have
+  /// a [textDirection] dependency), it is required for calculation of path to clip
+  ///
+  /// Used by [InkSplash] and [InkRipple], for example.
```

Comment:
- Prefer full sentences
```suggestion
  /// For examples on how the function is used, see [InkSplash] and [InkRipple].
```